### ISSUE TITLE
table migration

### ISSUE THREAD

**Issue by FantasticLBP** [Tencent/wcdb#581]
### The language of WCDB

> Objective-C 

### The version of WCDB

> latest version

### The platform of WCDB

> iOS

### The installation of WCDB

>Cocoapods

### What's the issue?

- version 1:
```Objective-C
@property (nonatomic, assign) NSInteger Age;
@property (nonatomic, copy)  NSString *name;
@property (nonatomic, copy) NSString *hobby;
//...
```

- version 2
```Objective-C
@property (nonatomic, assign) NSInteger nianling;
@property (nonatomic, copy)  NSString *name;
@property (nonatomic, copy) NSString *hobby;
//...
```
 I used model like version 1 to store data. Then I change the table structure like version 2. So I am curious the WCDB can migrate the data of Age to nianling?

<img width="1037" alt="image" src="https://user-images.githubusercontent.com/20414278/62961688-cf2bb800-be2f-11e9-9ff6-7e08a16c1108.png">

**Comment by FantasticLBP** [comment]
Relam can use migrationBlock to migrate the data. Does WCDB support this kind of migrate?

User (int age, string name) --> User(int Age, string name). The data of age can be migrated to Age?

**Comment by RingoD** [comment]
Yes, you can do this similar. See `WCDB_SYNTHESIZE_COLUMN` macro in the ORM chapter of the tutorial.

**Comment by FantasticLBP** [comment]
> Yes, you can do this similar. See `WCDB_SYNTHESIZE_COLUMN` macro in the ORM chapter of the tutorial.

In your opinion, I have a try. 
Before
```Objective-C
@property (nonatomic, assign) NSInteger age;
```
After
```Objective-C
@property (nonatomic, assign) NSInteger NianLing;
````

I used version 1 model to store data, and then I migrate property like version 2 used WCDB_SYNTHESIZE_COLUMN(Persom, NianLing, "age").  I stored some data and open the DB file, I found WCDB_SYNTHESIZE_COLUMN api only indicate NianLing is  releated to age. 

 When I store some data, age rows can be added  and  NianLing is empty. So can I think WCDB_SYNTHESIZE_COLUMN is only the specification between age and NianLing?


<img width="1037" alt="image" src="https://user-images.githubusercontent.com/20414278/63400445-6c758600-c406-11e9-8a01-a38b1f363135.png">

<img width="1050" alt="image" src="https://user-images.githubusercontent.com/20414278/63400846-d3476f00-c407-11e9-9516-0142fddc360d.png">
